comment.go

```diff
@@ -0,0 +1,31 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+package quicktest
+
+import "fmt"
+
+// Commentf returns a test comment whose output is formatted according to
+// the given format specifier and args. For instance:
```

Comment:
I'd probably add:

    It may be provided as the last argument to any check or assertion and
    will be displayed if the check or assertion fails.

?

Reply:
Done.